peerscout_api/main.py

```diff
@@ -131,86 +142,151 @@ def get_person_details_from_bq(
 
     query_job = client.query(sql)
     results = query_job.result()
-    return [row for row in results]
+    return list(results)
 
 
-def format_details_for_html(details: list) -> str:
-    formated_detail = ''
-    for detail in details:
-        formated_detail += "<br />" + detail
-    return formated_detail
-
-
-def get_formated_person_details_for_html(
-    person_ids: list,
-    is_person_recommended: bool
-) -> str:
+def get_person_details_from_bq(
+    person_ids: list
+):
     PROJECT_NAME = 'elife-data-pipeline'
     DATASET_NAME = get_deployment_env()
 
-    result_person_details = get_person_details_from_bq(
+    return query_bq_for_person_details(
         project=PROJECT_NAME,
         dataset=DATASET_NAME,
         person_ids=person_ids
     )
 
-    if is_person_recommended:
-        person_details = (
-            [
-                '<p>'
-                + person.person_name + '<br />' + person.institution + ', ' + person.country
-                # limited availability
-                + ('<br /><span style=\'color:red;\'><strong>!</strong></span> Limited availability: '
-                    + person.availability if person.availability else '')
-                # urls
-                + '<br /><a href=' + person.Website_URL + '>Website</a> | <a href='
-                + person.PubMed_URL + '>PubMed</a>'
-                # stats for initial submission
-                + ('<br />Days to respond: ' + person.days_to_respond
-                    if person.days_to_respond else '')
-                + ('; Requests: ' + person.requests if person.requests else '')
-                + ('; Responses: ' + person.responses if person.responses else '')
-                + ('; Response rate: ' + person.response_rate + '%' if person.response_rate else '')
-                # stats for full submission
-                + ('<br />No. of current assignments: ' + person.no_of_assigments
-                    if person.no_of_assigments else '')
-                + ('; Full submissions in 12 months: ' + person.no_full_submissions
-                    if person.no_full_submissions else '')
-                + ('; Decision time: ' + person.decision_time + ' days'
-                    if person.decision_time else '')
-                + '</p>'
-                for person in result_person_details
-            ]
-        )
-    else:
-        person_details = (
-            [
-                person.person_name + '; ' + person.institution + ', ' + person.country
-                for person in result_person_details
-            ]
-        )
 
-    return format_details_for_html(details=person_details)
+def get_html_text_for_recommended_person(
+    person: PersonProps
+) -> str:
+    return (
+        person.person_name
+        + ('<br />' if person.institution else '')
+        + (person.institution if person.institution else '')
+        + ((', ' + person.country) if (person.country and person.institution) else '')
+        # limited availability
```

Comment:
very minor: comments like this are often an opportunity to extract some code to a function